runtime/oti/vm_api.h

```diff
@@ -3115,10 +3117,13 @@ getStringUTF8Length(J9VMThread *vmThread,j9object_t string);
 *
 * @param[in] *vmThread current thread
 * @param[in] string the class name string
+* @param[in] allowedBitsForClassName the allowed bits for a valid class name,
+*            including CLASSNAME_INVALID, CLASSNAME_VALID_NON_ARRARY, CLASSNAME_VALID_ARRARY, or CLASSNAME_VALID.
```

Comment:
INVALID isn't actually valid here, as it's filtered out by findClassString.

Reply:
Removed `CLASSNAME_INVALID`.